src/kinetics/ReactionFactory.cpp

```diff
@@ -144,6 +144,15 @@ ReactionFactory::ReactionFactory()
         }
         return R;
     });
+
+    // register electron temperature reactions
+    reg("electron-temperature", [](const AnyMap& node, const Kinetics& kin) {
+        ETempReaction* R = new ETempReaction();
+        if (!node.empty()) {
+            setupETempReaction(*R, node, kin);
+        }
```

Comment:
Should be handled by a new constructor that takes `node` and `kin` as arguments.